src/NJsonSchema.CodeGeneration.CSharp/Models/EnumTemplateModel.cs

```diff
@@ -51,7 +51,18 @@ public IEnumerable<EnumerationItemModel> Enums
         {
             get
             {
+             
                 var entries = new List<EnumerationItemModel>();
+                
+                Dictionary<string, object> Descriptions = new Dictionary<string, object>();
+                Dictionary<string, object> Titles = new Dictionary<string, object>();
+
+                if (_schema.ExtensionData.ContainsKey("elaboration"))
+                    Descriptions = (Dictionary<string, object>)_schema.ExtensionData["elaboration"];
+
+                if (_schema.ExtensionData.ContainsKey("options"))
```

Comment:
Why "options"? Is this in the JSON Schema standard?

Reply:
I'm not familiar with the JSON schema. I am processing a lot of JSON files and see these two 'annotations' all over the text.

Reply:
examples  https://github.com/ANNA-DSB/Product-Definitions/tree/master/PROD/JSON